scss/_custom-styles.scss

```diff
@@ -132,3 +132,64 @@
     }
   }
 }
+
+// >> Arizona wordmark header.
+
+.arizona-header {
+  position: relative;
+}
+.block-a-line-logo {
+  width: auto;
+  height: 36px;
+  margin: 5px 20px 0 10px;
+}
+.arizona-line-logo {
+  width: 211px;
+  height: 16px;
+  margin: 17px 20px 17px 10px;
```

Comment:
Is there a reason the left and right margin values are different?  I might be missing something obvious but it seems to me that it'd be better to have the left margin be 20px as well (?).

Reply:
I agree, the left margin should also be 20px to give the logo even spacing.